compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py

```diff
@@ -113,7 +117,9 @@ def run(self):
 
         # combine ismip6 forcing data covering different periods
         # into a single file
-        input_file_list = self._files[period_endyear][model][scenario]
+        input_file_list = self._files[period_endyear][model][scenario] \
+                          [res_ismip6]
```

Comment:
You can't split indexing like that across lines unfortunately.  Try:
```suggestion
        input_file_list = \
            self._files[period_endyear][model][scenario][res_ismip6]
```

Reply:
Hmm, maybe it's allowed but I'd still prefer to avoid it.